pantheon/src/main/java/tech/pegasys/pantheon/cli/StandaloneCommand.java

```diff
@@ -37,13 +37,14 @@
 
   // Genesis file path with null default option if the option
   // is not defined on command line as this default is handled by Runner
-  // to use mainnet json file from resources
-  // NOTE: we have no control over default value here.
+  // to use mainnet json file from resources as indicated in the
+  // default network option
+  // Then we have no control over genesis default value here.
   @CommandLine.Option(
     names = {"--private-genesis-file"},
     paramLabel = MANDATORY_FILE_FORMAT_HELP,
     description =
-        "The path to genesis file. Setting this will also override --chain option to be CUSTOM"
+        "The path to genesis file. Setting this option makes --chain option ignored and requires --network-id to be set."
```

Comment:
We seem to have come back to calling this `--genesis-file` instead of `--private-genesis-file` but doesn't need to be changed as part of this PR.

Also I think the description should be referring to the `--network` option not the `--chain` option.

Reply:
The first point will be handled in another PR.
But I will change the desc for --network

Reply:
Changed in this PR finally as it was a few seconds to change it.